lib/scripts/behaviour.js

```diff
@@ -215,25 +215,26 @@ var dw_behaviour = {
      * avoids overeager security bans (see #3363)
      */
     securityCheck: function () {
-        var $checkA = jQuery('#security__check');
-        if (!$checkA.length) return;
-        if (sessionStorage.getItem('security-check')) {
+        var $checkDiv = jQuery('#security__check');
+        if (!$checkDiv.length) return;
+        if (sessionStorage.getItem('dw-security-check:' + DOKU_BASE)) {
```

Comment:
Could DOKU_BASE contain e.g. Asian or Russian characters that needs more attention? Or is everything utf-8 or something like that and handled well?

Reply:
I couldn't find anything about limits for the storage key. I assume it's internally implemented as a hash map with the key being hashed so it's format or encoding shouldn't matter at all. And JavaScript is generally UTF-8 safe everywhere.